stripe-ui-core/src/main/java/com/stripe/android/uicore/text/Html.kt

```diff
@@ -228,6 +231,56 @@ fun Html(
     urlSpanStyle: SpanStyle = SpanStyle(textDecoration = TextDecoration.Underline),
     imageAlign: PlaceholderVerticalAlign = PlaceholderVerticalAlign.AboveBaseline,
     onClick: () -> Unit = {}
+) {
+    val context = LocalContext.current
+    HtmlWithCustomOnClick(
```

Comment:
Is there any way to way some tests for these Html composables?

Reply:
Added some screenshot tests

Reply:
beautiful 👏